packages/filefeeds/src/filefeeds.ts

```diff
@@ -153,6 +154,16 @@ export class OneSchemaFileFeedsClass extends EventEmitter {
 
     this.#launchParams = launchParams
 
+    try {
+      this.#resumeTokenKey = `OneSchemaFileFeeds-${this.#params.userJwt}`
```

Comment:
Can we add a `-session-` in between the prefix and the jwt value, to keep it similar to the Importer keys?